doc/man/man3/crypto_chacha20_encrypt.3monocypher

```diff
@@ -273,9 +268,9 @@ crypto_chacha20_encrypt(&ctx,
 crypto_chacha20_set_ctr(&ctx, 0);
 crypto_chacha20_encrypt(&ctx, cipher_text, plain_text, 3 * 64);
 /* Wipe secrets if they are no longer needed */
-crypto_wipe(key,        sizeof(key));
+crypto_wipe(key,        32);
 crypto_wipe(&ctx,       sizeof(ctx));
-crypto_wipe(plain_text, sizeof(plain_text));
```

Comment:
CuleX: "I'm certain that we've agreed some time ago to avoid sizeof for the examples"